docs/v2/examples/flags.md

```diff
@@ -230,6 +230,52 @@ That flag can then be set with `--lang spanish` or `-l spanish`. Note that
 giving two different forms of the same flag in the same command invocation is an
 error.
 
+#### Multiple Values per Single Flag
+
+Using a slice flag allows you to pass multiple values for a single flag; the values will be provided as a slice:
+
+- `Int64SliceFlag`
+- `IntSliceFlag`
+- `StringSliceFlag`
+
+<!-- {
+  "args": ["&#45;&#45;greeting Hello", "&#45;&#45;greeting Hola"],
+  "output": "Hello, Hola"
+} -->
+
+```go
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/urfave/cli/v2"
```

Comment:
This needs to be v3 if we need to merge this PR to main. If you want this change in v2 can you rebase your changes off of v2-maint branch ?

Reply:
Ah, totally overlooked all of that, makes sense!

Reply:
So it'd be useful to have for both v2 and v3?

Reply:
Yes. But we would need separate PRs for each